src/scripts/interface/cellHighlightRenderer.ts

```
@@ -1,23 +1,50 @@
+    Elements.findById<HTMLCanvasElement>('cell-canvas').getContext('2d')!;
   private readonly _game: GameInfo;
+  private static readonly cardinalDirections: ReadonlyArray<RVector2> = [
+    new Vector2(0, 1),
+    new Vector2(1, 0),
+    new Vector2(0, -1),
+    new Vector2(-1, 0),
+  ];
+
+  private static getCardinalDirection(from: RVector2, to: RVector2): RVector2 {
+    if (from.x < to.x) {
+      return this.cardinalDirections[1];
```

Biome (1.9.4)

[error] Using this in a static context can be confusing.

this refers to the class.
Unsafe fix: Use the class name instead.


(lint/complexity/noThisInStatic)